a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -445,10 +469,30 @@ def _get_patched_ve_ip(self, ve_ip):
         return a10_utils.merge_host_and_network_ip(self._subnet.cidr, ve_ip)
 
     @device_context_switch_decorator
-    def delete_device_vlan(self, vlan_id, subnet_id, device_id=None, default_device_id=None, project_id=None):
+    def check_ve_ip_exists(self, vlan_id, config_ve_ip):
+        if self.axapi_client.vlan.exists(vlan_id):
+            if self.axapi_client.interface.ve.exists(vlan_id):
+                ve = self.axapi_client.interface.ve.get(vlan_id)
+                if config_ve_ip == 'dhcp':
+                    if ve.get('ve').get('ip') and ve.get('ve').get('ip').get('dhcp'):
+                        return True
+                    return False
+                else:
+                    if ve.get('ve').get('ip') and ve.get('ve').get('ip').get('address-list'):
```

Comment:
`ve` and `ip` keys are used everywhere here. Also it's required to have `ve` and `ip` keys present.

So for readability, just defined a variable such that `ve_ip = ve['ve'].get('ip')` then check if it exists and otherwise return false.